cpp/src/gandiva/engine.h

```diff
@@ -54,6 +54,13 @@ class GANDIVA_EXPORT Engine {
     functions_to_compile_.push_back(fname);
   }
 
+  /// Set BaseObjectCache.
+  template <class KeyType>
+  Status SetLLVMObjectCache(GandivaObjectCache<KeyType>& object_cache) {
+    execution_engine_->setObjectCache(&object_cache);
+    return Status::OK();
```

Comment:
why did you remove the error detection code?

Reply:
It was failling on appveyor with the message: `error C2039: 'hasError': is not a member of 'llvm::ExecutionEngine'`

Reply:
Ok. looks like earlier versions didnot contain this method. Also looking at setObjectCache impl it just sets the member variable so no need to check for error

Reply:
Ok, so I will just leave it withouht the error checking for compability with older versions